lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -52,7 +57,59 @@ module Style
       #       puts 1
       #     end
       #   end.freeze
+      #
+      # @example
+      #   # Magic comment - shareable_constant_value: literal
+      #
+      #   # bad
+      #   CONST = [1, 2, 3]
+      #
+      #   # good
+      #   # shareable_constant_value: literal
+      #   CONST = [1, 2, 3]
+      #
+      # NOTE: This special directive helps to create constants
+      # that hold only immutable objects, or Ractor-shareable
+      # constants. - ruby docs
+      #
```

Comment:
@bbatsov - Added some examples. Also added an excerpt from ruby docs so that shareable_constant_value doesn't turn out to be an alternative to **.freeze**